### ISSUE TITLE
Docker images for CUDA variants

### ISSUE THREAD

**Issue by sharvil** [pytorch/serve#1793]
### 🚀 The feature

It would be really nice to have Docker images on the Hub for the various releases of CUDA – in particular, the same CUDA versions as the official PyTorch repository (currently 10.2, 11.3, 11.6).

### Motivation, pitch

Older CUDA versions don't fully support newer hardware, e.g. CUDA 10.2 doesn't support compute capability 8.6 devices. While it's possible to build Docker images from the supplied Dockerfiles, it would be much quicker to get up and running if CUDA-specific images were pushed to Docker Hub to build our own images off of.

**Comment by maaquib** [comment]
@sharvil We do support [building from source](https://github.com/pytorch/serve/blob/master/docker/build_image.sh#L66-L82) for different versions of cuda. You can check out [this](https://github.com/pytorch/serve/tree/master/docker#create-torchserve-docker-image) for more details

**Comment by sharvil** [comment]
@maaquib in the "Motivation" section I pointed out that it's possible to build from source but is inconvenient. This feature request is to build and push the image variants to Docker Hub. Can you please reopen this feature request?

**Comment by msaroufim** [comment]
Hi @sharvil it's not difficult to do this, I do appreciate that you're suggesting only 3 versions that way we can deprecate old ones as needed. Historically our release process involves manual promotions which has made doing something like this a pain but once this is merged #1666 your ask will be easy to handle
